### DISCUSSION TITLE
Can't connect anymore with subversion

### DISCUSSION THREAD

**Discussion by roger21** [community/community#86785] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey there,

All of a sudden I can't connect to my repos with subversion, I get this error when I svn update:
`svn: E170013: Unable to connect to a repository at URL 'https://github.com/roger21/image-zoom.git/trunk'`
`svn: E160013: '/roger21/image-zoom.git/trunk' path not found`

So any clue what is happening? Or what should I check?

Also same problem with svn chechout.

And no problem with git cloned repos tho.

**Comment by roger21** [comment, marked as answer]
hummm maybe that ...

https://github.blog/2023-01-20-sunsetting-subversion-support/

**Comment by diemoschwarz** [comment]
That's the worst news of the last 5 years for us.  Our whole R&D activity relies on svn integrating a few open source components from github as svn:externals.

**Comment by imilne** [comment]
Yep, really sucks ass, and I never knew about it until everything stopped working :(

**Reply by Anobium** [reply]
Same here.

Huge amount of rework here.  

At what point did they NOT know that I ( we ) were using the SVN interface and they decided NOT to contact us!

0 out of 10 for thinking of us!

**Comment by maravento** [comment]
workaround:

```
sudo apt install -y python-is-python3
wget https://raw.githubusercontent.com/maravento/vault/master/scripts/python/gitfolderdl.py
chmod +x gitfolderdl.py
# Now Replace "my_user", "my_repo" and "my_folder". Example:
python gitfolderdl.py https://github.com/my_user/my_repo/my_folder
```
I hope you find it useful

**Comment by diraneyya** [comment]
For the workflows that used to rely on SVN for downloading a part of a repository (e.g. download a specific folder, without cloning and hence without history information), this is a possible solution:
- https://gist.github.com/diraneyya/c2171c2a262ac53f0d296c84d9d72e5c
- https://medium.com/@orwa.diraneyya/github-drops-svn-support-and-breaks-legacy-software-2a30bedbd08d
